tests/integration/targets/hcloud_iso_info/aliases

```diff
@@ -0,0 +1,2 @@
+cloud/hcloud
+shippable/hcloud/group2
```

Comment:
I suggest `group3`, its currently the emptiest -> does not cause longer CI runs.

```suggestion
shippable/hcloud/group3
```